tasks/install-ubuntu.yml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
This is identical to `install-debian.yml`. Can we consolidate, e.g. by using `ansible_pkg_mgr`?